fish-pepper/run-java-sh/fp-files/debug-options

```diff
@@ -6,7 +6,11 @@
 debug_options() {
   if [ "x${JAVA_ENABLE_DEBUG}" != "x" -o "x${JAVA_DEBUG_ENABLE}" != "x" -o "x${JAVA_DEBUG}" != "x" ]; then
     local debug_port=${JAVA_DEBUG_PORT:-5005}
-    echo "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=${debug_port}"
+    local suspend_mode="n"
+    if [ "x${JAVA_DEBUG_SUSPEND}" != "x" ]; then
```

Comment:
I would also check for `true` / `false` or `on` / `off`, `y`/`n` ... , not only for an empty value.

Otherwise you would then _enable suspend mode_ when you specify `JAVA_DEBUG_SUSPEND=n`.